File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/endo/END1001U02GrdPurposeInfo.java
package nta.med.data.model.ihis.endo;

public class END1001U02GrdPurposeInfo {
	private String n         ;
	private String codeName ;
	public END1001U02GrdPurposeInfo(String n, String codeName) {
		super();
		this.n = n;
		this.codeName = codeName;
	}
	public String getN() {
		return n;
	}
	public void setN(String n) {
		this.n = n;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Endo/END1001U02GrdPurposeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Endo
{
    [Serializable]
    public class END1001U02GrdPurposeInfo
    {
        private String _n;
        private String _codeName;

        public String N
        {
            get { return this._n; }
            set { this._n = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public END1001U02GrdPurposeInfo() { }

        public END1001U02GrdPurposeInfo(String n, String codeName)
        {
            this._n = n;
            this._codeName = codeName;
        }

    }
}